source/development_process/2_deployment.rst

```diff
@@ -150,7 +151,7 @@ kitchen with a cookbook, run the following:
 `kitchen init` will add a .kitchen.yml file, a .kitchen directory, and a test
 directory. The .kitchen.yml file specifies how to create a given virtual
 machine and which recipes to converge it with. Kitchen is configured for the
-project, the following commands can be used:
+project; the following commands can be used:
```

Comment:
`Kitchen is configured for the project;` seems like a silly sentence. Can we just get rid of it entirely?